client/a8c-for-agencies/sections/partner-directory/agency-details/hooks/use-details-form.ts

```diff
@@ -0,0 +1,42 @@
+import { useCallback, useState } from 'react';
+import { AgencyDetails } from 'calypso/a8c-for-agencies/sections/partner-directory/types';
+
+export default function useDetailsForm() {
+	const [ formData, setFormData ] = useState< AgencyDetails >( {
+		name: '',
+		email: '',
+		website: '',
+		bioDescription: '',
+		logoUrl: '',
+		landingPageUrl: '',
+		country: '',
+		isAvailable: true,
+		industry: '',
+		languagesSpoken: [],
+		budgetLowerRange: 0,
+		services: [],
+		products: [],
+	} );
+
+	const isValidFormData = useCallback(
```

Comment:
I realized that we don't need to make this a function but just a boolean. Can we change this and the Expertise form to make this a boolean?